lib/chef/provider/aws_route_table.rb

```diff
@@ -7,7 +7,7 @@ def action_create
     route_table = super
 
     if !new_resource.routes.nil?
-      update_routes(vpc, route_table)
+      update_routes(vpc, route_table, [new_resource.ignore_route_targets].flatten)
```

Comment:
As I referenced in your [other review](https://github.com/chef/chef-provisioning-aws/pull/151/files#r30183218) you can coerce the `ignore_route_targets` attribute in the resource so you always know it will be an array in here